bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/EventTriggersTypeDefinition.json

```diff
@@ -110,6 +110,41 @@
                "reference":"itemStateChangeTriggerID.event"
             }
          ]
+      },
+      {
+         "uid":"ThingEventTrigger",
+         "label":"Thing Trigger Event Trigger",
+         "description":"This triggers a rule if a channel emits a trigger",
+         "configDescriptions":[
+            {
+               "name":"channelId",
+               "type":"TEXT",
+               "context":"channel",
+               "label":"channel ID",
+               "description":"the id of the channel which should be observed for triggers",
+               "required":true
+            }
+         ],
+         "children":[
+            {
+               "id":"thingTriggerEventId",
+               "type":"GenericEventTrigger",
+               "configuration":{
+                  "eventSource":"$channelId",
+                  "eventTopic":"smarthome/channels/*/triggered",
+                  "eventTypes":"ThingTriggerEvent"
+               }
+            }
+         ],
+         "outputs":[
+            {
+               "name":"event",
```

Comment:
Again, I see that this is done in the existing modules this way, too, but it is something I already wanted to have changed since a long time.
I would very much prefer if the output of this trigger would be a "Type", so exactly what the handler emits (e.g. "TRIGGERED"). This would make it much nicer for following rule actions to use it. WDYT?